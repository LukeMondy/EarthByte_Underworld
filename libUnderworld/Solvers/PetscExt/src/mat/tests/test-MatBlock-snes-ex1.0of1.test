EXEC 		= ./test-MatBlock-snes-ex1.app
NPROCS 		= 1
TEST_TYPE       = tolerance_diff_pl,1.0e-10
