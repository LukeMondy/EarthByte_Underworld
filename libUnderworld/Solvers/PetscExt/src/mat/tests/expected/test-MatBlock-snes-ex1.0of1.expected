

========================= Assembled system =========================

  0 SNES Function norm 6.041522986797e+00 
  1 SNES Function norm 4.786756692342e+00 
  2 SNES Function norm 2.986456323228e+00 
  3 SNES Function norm 2.306244667700e-01 
  4 SNES Function norm 1.936314002653e-03 
  5 SNES Function norm 1.435593558383e-07 
  6 SNES Function norm 4.440892098501e-16 
number of Newton iterations = 6



========================= Block system =========================

  0 SNES Function norm 6.041522986797e+00 
  1 SNES Function norm 4.786756692342e+00 
  2 SNES Function norm 2.986456323228e+00 
  3 SNES Function norm 2.306244667699e-01 
  4 SNES Function norm 1.936314002654e-03 
  5 SNES Function norm 1.435593558383e-07 
  6 SNES Function norm 4.440892098501e-16 
number of Newton iterations = 6

